src/main/java/de/tum/in/www1/artemis/domain/participation/ParticipationVCSAccessToken.java

```diff
@@ -0,0 +1,59 @@
+package de.tum.in.www1.artemis.domain.participation;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import jakarta.persistence.UniqueConstraint;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.domain.DomainObject;
+import de.tum.in.www1.artemis.domain.User;
+
+/**
+ * A Participation.
+ */
+@Entity
+@Table(name = "participation_vcs_access_token", uniqueConstraints = { @UniqueConstraint(columnNames = { "user_id", "participation_id" }) })
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
```

Comment:
```suggestion
@JsonInclude(JsonInclude.Include.NON_EMPTY)
```
Do we even need the Json annotation, if we don't send objects of this class via Rest calls?

Reply:
Good point, I removed it